client/components/confirmDialog/confirmDialog.directive.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+angular.module('observatory3App')
+.controller('confirmDialogCtrl', function($scope,$uibModalInstance,msg){
```

Comment:
i know we've made files like this elsewhere but this isn't actually a directive, it's a controller so might want to rename the file.